OpenQA/Qemu/BlockDev.pm

```diff
@@ -186,13 +185,8 @@ sub _from_map ($self, $drives, $snap_conf) {
     $self->backing_file(OpenQA::Qemu::BlockDev->new()->_from_map(\@rest, $snap_conf))
       if @rest > 0;
 
-    return $self->driver($this->{driver})
-      ->file($this->{file})
-      ->node_name($this->{node_name})
-      ->size($this->{size})
-      ->needs_creating($this->{needs_creating})
-      ->implicit($this->{implicit})
-      ->snapshot($snap_conf->get_snapshot(sequence => $this->{snapshot}));
+    return $self->driver($this->{driver})->file($this->{file})->node_name($this->{node_name})->size($this->{size})->needs_creating($this->{needs_creating})
```

Comment:
this looks particularily ugly from the linter perspective

Reply:
I think these changes come due to `-fnl # don't remove new lines` being deleted.

Reply:
Yeah, I think we should keep `-fnl`

Reply:
Bringing back `-fnl` yields: no changes :tada: